app/components/reports/region_tree_component.html.erb

```diff
@@ -0,0 +1,25 @@
+<ul class="access-item-wrapper">
+  <% children.each do |parent, children| %>
+  <li>
+    <div class="access-item collapsed <%= parent.region_type %>">
+      <% depth(parent).times do %>
```

Comment:
Can we simply pass the depth into the recursive function call? That'll keep the view code _extremely_ agnostic of our region structure, right?

Reply:
We can, but I think I'd like to wait until I tackle the non India region hierarchy before changing this much more...as I think this will change depending on how I implement that.